### DISCUSSION TITLE
在hostloc nodeseek 小众软件都刷到了，是个老人了

### DISCUSSION THREAD

**Discussion by dianso** [jason5ng32/MyIP#42] [General]
在hostloc nodeseek 小众软件都刷到了，是个老人了

十几年前记得还和对象分别有禅猫博客，恍然如隔世什么的两个博客，记不得太清了，但是还黑过，但也没做破坏。

至于可能吧，极客公园，那是后话了

**Comment by jason5ng32** [comment]
确实老人了
